profiles/source_profile.go

```diff
@@ -429,12 +431,80 @@ func NewSourceProfileConnection(source string, params map[string]string) (Source
 	return conn, nil
 }
 
+type DirectConnectionConfig struct {
+	Host     string `json:"host"`
+	User     string `json:"user"`
+	Password string `json:"password"`
+	Port     string `json:"port"`
+	DbName   string `json:"dbName"`
+}
+
+type DatastreamConnProfile struct {
+	Name     string `json:"name"`
+	Location string `json:"location"`
+}
+
+type DataflowConfig struct {
+	Location      string `json:"location"`
+	Network       string `json:"network"`
+	Subnetwork    string `json:"subnetwork"`
+	HostProjectId string `json:"hostProjectId"`
+}
+
+type DataShard struct {
+	DataShardId           string                `json:"dataShardId"`
+	SrcConnectionProfile  DatastreamConnProfile `json:"srcConnectionProfile"`
+	DestConnectionProfile DatastreamConnProfile `json:"destConnectionProfile"`
```

Comment:
nit: what is the convention in HB. Is it to use Dst or Dest ?

Reply:
Its actually `Dst`. I've renamed this object to match the convention.